flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -227,6 +234,100 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
     }
   }
 
+  private fun setUser(user: Map<String, Any?>?, result: Result) {
+    Sentry.configureScope { scope ->
+      if (user == null) {
+        scope.user = null
+      } else {
+        val userInstance = User()
+
+        (user["email"] as? String)?.let { userInstance.email = it }
```

Comment:
Wondering if it's possible to use the `User#deserialize`, the difference is, that we have a `Map` and it requires a `JsonObjectReader`, but I guess we could spare some boilerplate by doing so, we'd need to serialize the `Map` tho.